Applications/YOLO/PyTorch/dataset.py

```diff
@@ -0,0 +1,66 @@
+##
```

Comment:
please add SPDX-License-Identifier: Apache-2.0 
as in https://github.com/nnstreamer/nntrainer/blob/6bf3610820817d298185d9dce8eda9d84a3a71ac/Applications/MNIST/PyTorch/main.py#L2

Reply:
I added this line. thanks!